internal/sdkv2provider/validators.go

```diff
@@ -74,3 +81,12 @@ func validateURL(v interface{}, k string) (s []string, errors []error) {
 	}
 	return
 }
+
+// validateZoneID ensures that the given Zone ID is valid.
+func validateZoneID(value string) error {
```

Comment:
this is a nice validator but a little overkill. we generally don't enforce this as at it's core, the value is just a hex string and no way to actually determine if it is a zone, account or other external identifier.

Reply:
Do you want it removed? I added it primarily for the import feature, as when you provide something that does not conform to the known format. Terraform terminates with an error and shows a message claiming the provider will be blamed for an error.

I can either drop this or move the validation into the import function. Thoughts?

Reply:
Done. Thank you!